README.md

```diff
@@ -52,6 +53,7 @@ The top level of the configuration file looks like this:
 ```
 * `__snapType` specifies the type of request Kombucha is going to issue. `__GET` is a simple HTTP `GET` request, and the remaining parameters are interpreted accordingly: `queryItems` are converted into key-value pairs and appended onto the `path`, etc.
    * The `snaps` entry above results in an HTTP `GET` to `https://httpbin.org/response-headers?foo=1&bar=2&baz=3`. In addition, Kombucha always sends an `Accept` header of `application/json`.
+   * The `snaps` entry also needs to contain the `nameIdentifier` key to uniquely identify the request. This name will be used to create a file with the snap result on disk.
```

Comment:
If we are going with `__snapName`, it might be good to emphasize its importance a little more by moving this description up to line `54` in the docs. ie. I think `__snapName` is probably (now that we’ve added it!) the most important parameter to explain!